taylor_min_distance.py

```diff
@@ -5,3 +5,4 @@ def minTimeToVisitAllPoints(self, points: List[List[int]]) -> int:
             dist += max(abs(x0 - x1), abs(y0 - y1))
 
         return dist
+#Commenting stuff because I'm crazy.
```

Comment:
What a shitty comment, please remove this garbage ASAP

Reply:
I never said it was good.